extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -27,10 +27,22 @@ package stackdriver.config.v1alpha1;
 import "google/protobuf/duration.proto";
 import "google/protobuf/wrappers.proto";
 
-// next id: 10
+// next id: 12
 message PluginConfig {
+  // Types of Access logs to export.
+  enum AccessLogging {
+    // No Logs.
+    NONE = 0;
+    // All logs including both success and error logs.
+    FULL = 1;
+    // All error logs. This is
+    // currently only available for outbound/client side logs.
+    ERR_ONLY = 2;
```

Comment:
super nit: slight preference for fully spelling this out `ERRORS_ONLY`.

Reply:
Sure..fixed!